### ISSUE TITLE
UserAccountsDrawerHeader.otherAccountsPictures touch target too small

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#19906]
Touch targets for the secondary accounts displayed in the top right of the UserAccountsDrawerHeader are too small. They are 40x40, but should be 48x48.

**Comment by jonahwilliams** [comment]
Fixed by https://github.com/flutter/flutter/pull/20266
